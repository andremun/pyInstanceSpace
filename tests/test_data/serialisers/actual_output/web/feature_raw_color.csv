Row,Max_Normalized_Entropy_attributes,Normalized_Entropy_Class_Attribute,Mean_Mutual_Information_Attribute_Class,ErrorRate_Decision_Node,WeightedDist_StdDev,Max_Feature_Efficiency_F3,Collective_Feature_Efficiency_F4,Training_Error_Linear_Classifier_L2,Fraction_Points_Class_Boundary_N1,Nonlinearity_Nearest_Neighbor_Classifier_N4
abalone,81,50,45,173,69,20,24,135,167,183
abalone_ori,81,120,72,91,69,131,141,18,33,199
appendicitis,70,40,20,59,21,76,192,103,120,118
asbestos,75,80,28,95,24,35,34,80,147,161
audiology_std,64,177,19,105,20,216,246,16,15,17
australian,77,39,11,201,32,8,13,75,113,99
auto1,8,30,0,114,15,0,0,134,190,140
auto2,81,25,0,159,15,0,1,185,255,0
auto4,80,44,0,215,14,2,13,150,205,0
auto5,75,66,3,153,7,45,217,93,155,0
auto6_1,81,121,7,118,15,14,107,65,121,29
auto6_2,77,117,5,117,17,11,70,65,124,33
auto6_3,79,128,11,129,11,40,207,65,122,19
auto7_1,80,89,5,149,23,3,5,104,165,195
auto7_2,76,65,3,206,27,16,20,173,245,200
auto7_3,75,95,4,152,17,7,15,104,158,177
auto8,75,144,6,100,3,43,253,0,101,0
automobile,140,124,48,138,24,130,254,86,76,75
bach,80,161,73,43,112,170,208,2,2,113
balance,55,54,18,192,17,0,0,30,96,111
banana,81,29,11,187,71,1,1,233,78,192
bankruptcy,65,69,35,105,22,171,254,254,83,39
blogger,79,52,6,119,25,0,0,166,131,167
blood,81,30,5,104,50,2,3,123,177,233
breast,95,40,5,129,28,5,5,152,196,182
breast_cancer,93,40,4,131,26,0,1,154,187,193
breast_cancer_wis_dia,72,36,68,35,106,31,59,17,23,22
breast_cancer_wis_dia_no_Nas,73,37,69,37,106,31,61,16,23,14
breast_cancer_wis_ori,80,39,41,29,47,135,254,25,28,7
breast_cancer_wis_pro2,99,38,3,104,23,46,254,123,181,131
breast_tissue,70,154,109,93,25,182,246,83,66,85
breast_tissue_merged,70,109,98,64,25,216,254,96,44,29
bupa,81,44,4,169,44,8,27,218,237,189
car,35,42,15,115,24,120,155,77,60,118
cardio10,79,105,56,79,70,119,187,47,32,90
cardio3,79,32,16,69,70,66,105,53,41,107
chess-krkp,3,31,2,210,52,47,110,32,112,72
chronic_kidney_disease,79,41,39,113,73,171,246,11,4,0
chronic_kidney_disease_full,79,41,39,126,73,171,246,11,4,0
cilinder,158,40,8,162,18,32,89,158,180,111
cilinder_no_NAs,163,44,4,157,18,15,104,185,192,108
climate,81,15,2,32,14,53,150,44,70,54
cnae9,14,126,2,81,71,32,254,23,19,20
congressional,12,41,37,47,40,0,0,17,46,12
congressional_no_Nas,16,48,36,63,40,0,0,15,63,9
connectionist_vowel,100,140,45,75,24,123,201,47,16,120
contraceptive,79,56,4,190,72,6,6,161,225,224
credit_no_Nas,95,40,13,167,30,8,16,70,111,84
diabetic,79,36,4,184,62,12,27,206,227,216
echocardio,78,56,30,22,16,255,254,28,71,18
echocardio_no_Nas,60,55,22,24,16,253,254,145,66,13
ecoli,73,104,66,108,35,166,211,40,28,80
fertility,70,29,1,49,13,5,12,62,110,151
firm,0,16,2,70,45,9,9,0,35,49
first,81,28,7,182,189,46,226,0,0,57
flare,45,93,33,139,78,120,157,59,56,122
forest,81,83,56,130,46,93,228,86,55,51
fourclass,79,35,25,144,21,62,70,111,2,139
gesture_va,81,64,20,143,158,7,25,104,98,182
grammatical_a1,81,35,22,69,54,34,254,87,118,105
grammatical_a2,80,28,42,20,65,133,254,21,31,24
grammatical_a3,77,34,52,32,52,45,254,43,70,74
grammatical_a4,81,27,23,54,53,76,254,56,56,80
grammatical_a5,80,36,24,96,53,38,254,68,83,86
grammatical_a6,81,27,34,27,63,129,254,29,43,47
grammatical_a7,81,23,20,67,61,44,255,33,58,72
grammatical_a8,78,36,30,46,55,20,254,58,67,61
grammatical_a9,81,34,34,36,57,43,254,46,71,59
grammatical_b2,79,28,16,100,87,25,254,68,67,120
grammatical_b3,80,35,46,50,52,54,254,46,78,85
grammatical_b4,81,34,22,53,51,38,254,35,69,74
grammatical_b5,79,34,24,178,56,31,254,89,80,92
grammatical_b6,80,29,18,75,81,30,254,45,55,78
grammatical_b7,81,27,26,42,61,70,254,42,74,87
grammatical_b8,77,35,28,53,47,73,254,45,63,52
grammatical_b9,81,33,21,91,54,21,254,63,100,118
haberman,75,37,5,125,24,7,8,137,222,200
hayes,54,81,28,247,18,21,117,173,157,131
heart_cleverland,75,88,17,138,25,29,79,64,114,152
heart_cleverland_no_Nas,76,89,17,138,25,30,79,65,115,150
heart_hungarian,76,43,28,93,10,180,218,77,59,3
heart_hungarian_no_Nas,99,117,27,39,10,183,229,208,81,0
heart_switzerland,67,114,11,166,15,54,119,104,163,160
heart_switzerland_no_Nas,68,145,16,224,15,108,202,104,161,101
heart_va,71,114,21,176,11,61,79,104,122,91
heart_va_no_Nas,96,171,30,211,11,119,247,122,194,86
hepatitis,77,37,11,80,22,60,144,100,116,55
hepatitis_no_Nas,76,38,9,86,22,88,251,84,128,66
hiv_746,108,39,33,124,30,5,11,179,184,173
hiv_impens,103,22,10,66,25,3,6,81,157,219
horse_colic_lesion,80,42,25,94,24,9,30,38,64,17
human,80,76,103,93,93,162,254,4,7,25
hv_no_noise,79,36,1,233,98,1,164,255,217,240
ilpd,79,35,7,127,54,29,58,149,204,199
ilpd_no_Nas,79,34,7,126,54,29,58,148,203,204
image,81,100,125,104,58,176,234,36,8,40
internet,64,16,0,48,127,15,221,14,23,60
ionosphere,81,42,22,78,60,50,253,60,94,87
iris,77,86,114,115,11,164,201,106,26,35
japanese,93,39,12,171,30,8,16,75,116,96
japanese_no_Nas,95,40,13,167,30,8,16,70,111,84
leaf,115,237,174,27,35,220,253,17,17,54
led,10,149,57,73,53,0,0,45,36,103
lenses,85,115,34,99,2,131,226,130,85,34
libras,81,186,94,51,25,156,254,31,21,89
liver,81,44,4,165,44,8,27,218,237,186
lsvt,81,50,10,96,24,79,255,57,150,31
lung,90,126,20,190,7,76,252,146,223,0
lung_no_NAs,88,133,25,138,7,113,242,128,198,0
lymphography,87,66,20,191,15,135,146,43,72,39
madelon,79,32,0,172,16,1,254,157,228,0
mammographics,77,37,28,79,55,12,11,93,140,111
mammographics_no_NAs,79,38,29,100,55,12,12,99,146,122
mechanical,79,67,9,158,126,37,41,79,122,242
mice,80,119,49,86,20,147,247,0,0,8
mice_no_Nas,80,131,58,84,20,171,247,4,1,7
molecular_promoter,70,57,8,155,9,37,36,44,163,0
molecular_splice,31,48,7,164,53,42,40,69,138,13
monks1,45,41,6,255,9,64,62,174,132,151
monks2,44,37,0,153,10,0,0,178,105,233
monks3,45,41,14,101,9,0,0,111,157,64
multiple,79,121,58,72,39,161,254,0,2,0
mushroom,50,27,28,97,132,58,253,27,0,50
onehund_mar,79,254,166,2,57,225,254,5,2,7
onehund_sha,79,254,255,2,76,226,254,5,4,73
onehund_tex,79,255,149,2,65,222,254,5,2,21
optical,69,106,44,68,61,62,128,5,1,8
ozone1_no_Nas,81,3,2,7,64,99,254,16,29,136
ozone8_no_Nas,81,9,4,25,64,66,222,36,52,138
page,80,17,21,30,147,98,109,20,10,103
pamap9,81,30,24,22,46,140,254,28,2,72
parkinson_speech,79,35,8,11,61,206,254,30,43,33
parkinsons,73,39,21,89,24,94,252,128,54,93
pen,81,98,99,69,118,102,142,11,0,39
phishing,13,26,6,200,214,34,33,38,35,53
phoneme,80,25,19,108,89,32,34,133,81,150
pima,81,36,9,156,53,1,5,182,180,157
pitt1_1,70,49,5,74,15,148,218,67,85,127
pitt1_2,70,68,28,87,15,175,224,66,63,71
pitt1_3,70,101,23,146,15,111,185,130,143,138
pitt1_4,70,94,18,155,15,121,181,95,123,136
pitt2_1,70,49,5,74,14,93,203,67,96,142
pitt2_2,70,68,29,101,14,147,223,48,62,74
pitt2_3,70,101,25,148,14,95,169,130,146,125
pitt2_4,70,94,19,142,14,94,157,95,135,121
planning,74,43,0,127,21,25,110,148,247,218
post,48,58,3,122,15,45,89,100,172,206
primary,113,41,4,154,22,0,0,122,176,148
qsar,81,34,9,114,78,20,85,101,111,83
ring,81,28,16,175,190,14,103,120,182,157
robot1,77,73,92,90,86,152,212,100,7,65
robot2,76,88,41,50,38,180,194,59,32,105
robot3,76,73,38,50,38,179,190,74,34,93
robot4,81,47,58,59,92,174,181,72,17,90
robot5,80,78,68,105,97,131,158,104,17,138
saheart,76,39,6,147,28,13,23,180,206,182
satimage,80,77,120,98,114,99,157,33,21,56
secom,79,9,2,3,0,164,254,34,62,3
seeds,71,81,105,122,11,169,233,74,32,34
seismic,79,8,2,23,90,15,24,34,64,225
soybean_large,79,188,73,67,33,219,243,18,13,15
soybean_large_no_Nas,81,183,65,85,33,208,239,27,19,18
soybean_small,133,142,90,101,9,178,214,13,25,0
spambase,80,28,10,117,136,23,97,87,71,75
spect_heart,15,33,5,89,49,37,53,107,138,177
spectf_heart,77,33,7,89,51,77,253,107,160,51
statlog_australian,93,39,13,172,31,8,16,75,114,94
statlog_german,77,32,2,133,43,3,6,146,178,148
statlog_heart,77,46,13,142,24,3,23,79,143,92
statlog_is,81,100,125,92,58,176,234,36,8,41
statlog_ls,80,77,120,98,114,99,157,33,21,55
statlog_vehicle,79,81,35,160,33,51,120,130,97,146
tae,77,86,7,217,24,18,28,173,178,200
tao,80,34,36,129,17,94,92,84,30,89
teaching,165,86,26,185,24,18,28,173,178,200
texture,80,111,157,63,83,170,236,28,0,16
thyroid_allbp,80,5,2,12,133,99,120,14,14,122
thyroid_allbp_no_Nas,79,6,2,15,133,103,119,16,15,125
thyroid_allhyper,80,3,2,5,133,211,219,5,5,93
thyroid_allhypo,80,12,3,27,133,195,206,20,24,176
thyroid_allrep,80,5,2,7,133,183,197,8,9,127
thyroid_allrep_no_Nas,79,4,1,5,133,187,202,7,8,131
thyroid_ann,81,10,4,4,255,242,244,25,32,136
thyroid_ann_no_Nas,81,10,4,4,255,242,244,25,32,136
thyroid_dis,80,0,0,0,133,34,118,8,10,173
thyroid_dis_no_Nas,79,0,0,1,133,38,97,9,11,182
thyroid_hypothyroid,79,5,4,5,0,52,83,24,17,88
thyroid_new_thyroid,71,59,64,85,41,160,205,104,27,37
thyroid_sick,80,7,1,21,133,36,76,31,25,101
thyroid_sick_no_Nas,79,10,2,30,133,40,68,41,33,102
tic,26,34,2,155,14,0,0,180,199,155
titanic,28,29,9,139,143,0,0,116,178,255
trains,255,120,39,86,7,131,203,52,206,0
turkiye,34,109,5,90,106,0,0,40,72,246
urban,77,128,73,83,30,159,254,26,35,38
user,78,92,39,94,19,142,172,130,72,57
vert2,75,41,18,98,29,93,114,167,128,114
vert3,75,70,50,52,29,123,161,120,103,86
wall2,80,53,120,45,131,200,229,83,3,20
wall24,80,53,48,78,85,110,245,87,47,117
wall4,80,53,80,136,153,200,254,83,11,59
wave,81,49,32,141,44,44,74,64,87,45
wave2,81,49,16,128,46,54,85,62,101,28
weight,79,63,178,128,15,226,253,0,0,0
wholesale,77,38,28,57,58,109,158,160,81,46
wilt,79,6,2,18,68,39,88,28,35,149
wine,74,82,67,149,17,180,251,36,31,9
wine_quality,80,56,9,168,162,39,58,73,72,225
winsconsin,73,37,69,35,106,31,61,16,23,21
wpbc,72,38,2,104,23,46,252,123,175,137
wpbc_no_Nas,73,39,2,103,23,47,253,123,174,138
zoo,73,144,78,112,27,193,232,19,18,10
