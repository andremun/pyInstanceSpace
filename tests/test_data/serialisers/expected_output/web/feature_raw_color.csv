Row,Max_Normalized_Entropy_attributes,Normalized_Entropy_Class_Attribute,Mean_Mutual_Information_Attribute_Class,ErrorRate_Decision_Node,WeightedDist_StdDev,Max_Feature_Efficiency_F3,Collective_Feature_Efficiency_F4,Training_Error_Linear_Classifier_L2,Fraction_Points_Class_Boundary_N1,Nonlinearity_Nearest_Neighbor_Classifier_N4
abalone,82,51,45,174,69,20,24,135,167,183
abalone_ori,82,120,72,91,69,131,142,19,33,200
appendicitis,71,40,21,59,22,77,192,103,120,119
asbestos,76,80,29,96,24,35,34,80,147,162
audiology_std,65,178,20,105,20,217,246,16,16,17
australian,77,40,12,201,32,9,13,75,114,99
auto1,8,30,0,114,16,0,0,135,191,141
auto2,81,25,0,160,16,0,2,185,255,0
auto4,80,44,1,215,14,2,13,150,205,0
auto5,76,66,4,154,7,45,218,93,155,0
auto6_1,81,122,7,119,16,15,108,65,121,30
auto6_2,77,118,6,117,17,11,70,65,125,33
auto6_3,79,129,11,130,11,41,208,65,122,19
auto7_1,81,90,6,150,24,4,6,104,165,195
auto7_2,77,65,3,206,28,17,20,173,245,200
auto7_3,76,95,5,152,17,8,16,104,158,178
auto8,76,145,7,100,4,44,254,0,102,0
automobile,141,125,48,139,24,130,254,87,77,75
bach,80,161,74,44,112,170,209,3,2,114
balance,56,55,18,192,17,0,0,30,97,112
banana,81,29,12,187,72,1,2,233,78,192
bankruptcy,66,70,35,105,23,171,255,255,84,40
blogger,79,52,7,119,25,0,0,166,132,168
blood,81,30,5,104,51,2,3,124,178,233
breast,95,41,5,130,29,6,6,152,197,183
breast_cancer,94,41,5,132,26,1,2,155,188,194
breast_cancer_wis_dia,73,37,68,36,107,31,59,18,24,23
breast_cancer_wis_dia_no_Nas,73,37,69,37,107,32,62,17,24,15
breast_cancer_wis_ori,80,39,42,29,48,135,255,26,29,7
breast_cancer_wis_pro2,100,39,3,104,23,46,255,124,181,131
breast_tissue,70,154,109,94,26,182,247,83,67,86
breast_tissue_merged,70,109,98,64,26,216,255,97,45,30
bupa,82,44,4,169,44,8,27,219,237,189
car,35,43,16,116,24,120,156,78,61,119
cardio10,79,105,56,79,70,120,187,48,32,90
cardio3,79,33,16,70,70,67,106,54,41,107
chess-krkp,3,32,3,211,53,48,110,32,113,72
chronic_kidney_disease,79,42,39,113,73,172,246,12,5,0
chronic_kidney_disease_full,79,42,39,126,73,172,246,12,5,0
cilinder,159,41,9,163,18,33,89,158,180,111
cilinder_no_NAs,164,44,4,157,18,15,105,186,192,108
climate,81,15,2,33,14,54,151,44,70,54
cnae9,14,127,2,82,71,32,254,23,20,21
congressional,12,42,37,48,40,0,0,18,46,12
congressional_no_Nas,17,49,36,64,40,0,0,16,63,10
connectionist_vowel,101,140,46,76,25,123,202,47,16,121
contraceptive,80,57,5,190,72,6,6,161,225,225
credit_no_Nas,95,40,13,168,30,9,17,71,111,84
diabetic,80,37,4,185,63,13,27,207,227,216
echocardio,78,57,31,23,17,255,255,28,72,19
echocardio_no_Nas,61,56,23,24,17,253,255,145,67,14
ecoli,74,104,66,109,35,167,211,41,29,80
fertility,71,29,2,49,13,5,13,62,111,151
firm,0,16,2,70,45,10,10,0,36,49
first,81,28,8,182,190,47,226,0,0,58
flare,45,93,33,139,79,120,157,60,56,122
forest,82,83,56,130,46,93,229,87,56,52
fourclass,79,36,26,144,21,63,71,111,3,140
gesture_va,81,65,21,144,159,8,26,104,98,183
grammatical_a1,81,36,23,70,55,35,255,87,119,106
grammatical_a2,81,29,43,21,65,134,255,21,31,24
grammatical_a3,78,34,53,32,53,46,255,44,71,74
grammatical_a4,81,27,23,54,53,76,255,57,57,80
grammatical_a5,80,37,25,96,53,38,255,68,84,86
grammatical_a6,81,28,35,28,63,129,255,30,43,48
grammatical_a7,81,24,20,67,62,45,255,34,58,73
grammatical_a8,78,36,30,46,55,20,255,59,68,62
grammatical_a9,81,34,34,37,57,44,255,47,72,59
grammatical_b2,80,29,17,100,88,26,255,69,68,120
grammatical_b3,80,35,47,50,52,54,255,46,78,85
grammatical_b4,82,35,22,53,51,38,255,35,70,75
grammatical_b5,79,35,25,179,57,31,255,90,81,92
grammatical_b6,81,29,18,76,81,31,255,46,56,79
grammatical_b7,81,27,26,43,62,70,255,43,75,87
grammatical_b8,78,35,29,53,48,73,255,46,64,53
grammatical_b9,82,34,22,91,55,22,255,63,100,118
haberman,75,38,6,126,25,8,8,138,223,200
hayes,54,82,28,248,19,21,118,173,157,132
heart_cleverland,76,89,18,138,26,30,79,65,115,152
heart_cleverland_no_Nas,76,90,18,139,26,30,80,65,115,151
heart_hungarian,76,44,29,93,11,181,219,78,59,4
heart_hungarian_no_Nas,100,117,27,40,11,183,229,208,82,0
heart_switzerland,68,114,11,167,15,55,119,104,163,160
heart_switzerland_no_Nas,69,145,17,225,15,108,203,104,161,102
heart_va,72,114,21,177,11,62,79,104,122,92
heart_va_no_Nas,96,172,30,212,11,119,247,122,194,86
hepatitis,77,38,12,81,22,61,145,101,117,56
hepatitis_no_Nas,77,38,9,87,22,88,252,85,129,66
hiv_746,109,39,33,124,30,6,11,179,185,173
hiv_impens,103,22,10,67,26,4,7,82,157,220
horse_colic_lesion,81,42,26,95,25,10,30,38,64,17
human,81,76,104,93,93,162,255,5,8,26
hv_no_noise,79,36,1,234,98,2,164,255,217,240
ilpd,80,35,7,127,55,29,58,149,205,200
ilpd_no_Nas,80,35,7,126,55,29,59,148,203,205
image,81,100,126,104,58,176,234,37,8,41
internet,65,17,1,48,127,15,222,15,23,60
ionosphere,82,42,22,78,60,50,254,61,95,88
iris,78,87,115,116,12,165,202,106,26,35
japanese,94,40,13,171,30,8,17,75,117,97
japanese_no_Nas,95,40,13,168,30,9,17,71,111,84
leaf,115,237,175,28,36,221,254,17,18,55
led,11,149,58,74,54,0,0,46,36,104
lenses,86,116,34,100,2,131,227,130,85,35
libras,81,187,95,52,25,157,254,32,22,89
liver,82,44,4,165,44,8,27,219,237,187
lsvt,82,51,11,97,24,79,255,58,151,31
lung,91,126,21,191,7,76,252,146,224,0
lung_no_NAs,88,134,25,139,7,113,242,128,199,0
lymphography,87,67,20,191,16,136,146,43,72,40
madelon,80,33,0,173,17,1,255,157,228,0
mammographics,77,38,29,80,56,12,12,94,140,112
mammographics_no_NAs,80,39,29,100,56,13,12,100,147,123
mechanical,80,67,10,159,127,37,42,80,122,243
mice,81,119,49,86,21,148,247,0,0,8
mice_no_Nas,81,131,58,84,21,171,248,4,1,7
molecular_promoter,70,58,8,155,9,37,36,44,163,0
molecular_splice,31,49,8,164,54,42,41,69,139,13
monks1,46,41,7,255,10,65,63,174,133,151
monks2,45,38,0,153,10,0,0,178,105,233
monks3,46,41,14,102,10,0,0,112,158,65
multiple,80,122,59,73,40,162,255,1,3,0
mushroom,50,28,28,98,132,58,254,28,1,51
onehund_mar,79,255,166,2,58,225,255,5,3,8
onehund_sha,79,255,255,2,76,226,255,5,4,74
onehund_tex,79,255,149,2,65,222,255,5,2,21
optical,69,107,44,69,61,62,128,6,1,8
ozone1_no_Nas,81,4,3,7,65,99,255,16,30,137
ozone8_no_Nas,81,10,4,25,65,67,222,36,52,139
page,81,17,22,30,148,99,109,20,11,103
pamap9,81,31,25,22,47,140,255,28,2,72
parkinson_speech,79,36,9,11,61,207,255,31,43,34
parkinsons,73,40,22,90,24,94,252,128,54,93
pen,81,99,99,69,118,102,142,12,0,40
phishing,14,27,7,200,215,35,34,38,35,54
phoneme,81,25,20,108,89,32,34,134,81,150
pima,81,36,10,156,53,2,6,182,181,157
pitt1_1,70,50,6,74,16,148,219,67,86,128
pitt1_2,70,68,29,87,16,175,225,66,63,72
pitt1_3,70,102,24,147,16,111,186,130,143,138
pitt1_4,70,94,18,155,15,121,181,95,124,137
pitt2_1,70,50,5,74,14,93,203,67,96,143
pitt2_2,70,68,29,101,14,147,223,48,62,75
pitt2_3,70,102,25,148,14,95,170,130,147,125
pitt2_4,70,94,19,142,14,95,158,95,135,121
planning,74,44,1,127,22,26,111,149,248,218
post,49,59,3,123,16,46,90,100,173,206
primary,114,42,4,155,22,0,0,123,177,149
qsar,81,34,9,114,79,21,86,102,111,83
ring,81,28,16,176,190,14,103,121,182,158
robot1,78,73,92,90,87,153,212,100,7,65
robot2,77,89,42,50,39,181,195,60,33,105
robot3,77,73,38,50,39,180,190,75,35,94
robot4,82,48,58,60,93,175,181,72,17,91
robot5,80,79,69,106,97,131,158,104,17,139
saheart,77,40,6,148,29,13,23,180,207,183
satimage,80,77,121,98,114,100,157,34,21,56
secom,80,10,2,4,0,164,255,35,63,3
seeds,72,81,105,122,12,170,233,74,33,35
seismic,80,9,2,24,90,16,25,34,65,226
soybean_large,79,188,73,68,33,219,244,18,14,15
soybean_large_no_Nas,81,183,66,85,33,208,240,27,19,18
soybean_small,133,142,90,101,10,178,214,14,25,0
spambase,80,29,11,117,137,24,98,88,72,76
spect_heart,16,34,6,90,49,37,53,107,139,177
spectf_heart,78,34,8,90,52,78,253,107,161,51
statlog_australian,94,40,14,173,31,9,17,75,115,95
statlog_german,77,33,2,133,43,4,6,146,178,149
statlog_heart,78,47,13,142,25,4,24,79,143,92
statlog_is,81,100,126,93,58,176,234,37,8,41
statlog_ls,80,77,121,98,114,100,157,34,21,56
statlog_vehicle,80,81,35,161,33,51,120,130,98,147
tae,78,87,8,217,24,19,28,173,179,200
tao,80,34,36,129,17,94,92,85,31,90
teaching,166,87,27,186,24,19,28,173,179,200
texture,81,112,157,64,83,171,237,28,1,16
thyroid_allbp,80,6,2,13,133,99,120,15,14,122
thyroid_allbp_no_Nas,79,7,2,15,133,104,120,17,16,126
thyroid_allhyper,80,3,2,5,133,211,219,6,5,93
thyroid_allhypo,80,12,4,28,133,195,206,20,24,177
thyroid_allrep,80,5,2,8,133,184,197,9,9,127
thyroid_allrep_no_Nas,79,4,2,6,133,187,203,7,8,132
thyroid_ann,81,10,4,5,255,242,245,25,32,137
thyroid_ann_no_Nas,81,10,4,5,255,242,245,25,32,137
thyroid_dis,80,0,0,0,133,35,118,8,10,174
thyroid_dis_no_Nas,79,1,0,1,133,38,98,9,11,183
thyroid_hypothyroid,79,6,4,5,0,53,83,25,18,89
thyroid_new_thyroid,71,59,65,85,42,161,205,105,27,38
thyroid_sick,80,8,2,21,133,36,77,32,26,101
thyroid_sick_no_Nas,79,11,2,30,133,41,69,42,34,103
tic,26,35,2,155,14,0,0,180,199,155
titanic,29,30,9,139,144,0,0,117,179,255
trains,255,121,40,87,8,131,204,52,207,0
turkiye,34,110,5,91,107,0,0,40,72,247
urban,77,128,73,83,31,160,255,26,36,39
user,79,93,40,95,19,143,173,130,73,58
vert2,75,41,19,99,29,94,114,168,129,115
vert3,75,71,51,52,29,124,162,120,103,87
wall2,81,53,121,45,132,201,229,84,4,21
wall24,81,53,48,78,86,111,245,88,48,118
wall4,81,53,81,136,153,201,254,83,12,60
wave,82,50,32,141,45,44,75,65,87,45
wave2,82,50,17,129,47,54,85,62,102,29
weight,80,64,178,128,15,226,254,1,0,0
wholesale,78,39,29,58,58,110,158,161,82,46
wilt,80,6,3,18,68,39,89,28,36,149
wine,75,83,67,149,17,181,251,36,31,10
wine_quality,81,56,9,168,162,39,59,74,72,225
winsconsin,73,37,69,35,107,32,62,17,24,21
wpbc,73,39,3,104,24,46,252,124,175,137
wpbc_no_Nas,73,39,3,103,24,47,254,123,174,139
zoo,73,145,79,113,27,193,232,20,18,10
